kubernetes/config/common.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: v1
+data:
+  common.libsonnet: |
+    {
+      blobstore: {
+        contentAddressableStorage: {
+          sharding: {
+            hashInitialization: 11946695773637837490,
+            shards: [
+              {
+                backend: { grpc: { address: 'storage-0.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+              {
+                backend: { grpc: { address: 'storage-1.storage.buildbarn:8981' } },
+                weight: 1,
+              },
+            ],
+          },
+        },
+        actionCache: {
+          sharding: {
+            hashInitialization: 14897363947481274433,
+            shards: [
+              {
+                backend: { grpc: { address: 'storage-0.storage.buildbarn:8981' } },
```

Comment:
Might be worth putting a comment here that if your `namespace` is different then the address will be 
`storage-0.storage.{namespace}:8981`.

Or even better, as it's already assumed these are deployed in same namespace, should we remove namespace from here?

Reply:
Yeah, it may be best to remove it altogether.

Reply:
```suggestion
                backend: { grpc: { address: 'storage-0.storage:8981' } },
```

Reply:
I have a local branch with these changes but the deployment wasn't working as I was pushing to get this merged. I haven't had the time to test this and make the necessary changes.